osu.Game/Localisation/EditorStrings.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class EditorStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.Editor";
+
+        /// <summary>
+        /// "File"
+        /// </summary>
+        public static LocalisableString File => new TranslatableString(getKey(@"file"), @"File");
+
+        /// <summary>
+        /// "Edit"
+        /// </summary>
+        public static LocalisableString Edit => new TranslatableString(getKey(@"edit"), @"Edit");
+
+        /// <summary>
+        /// "Undo"
+        /// </summary>
+        public static LocalisableString Undo => new TranslatableString(getKey(@"undo"), @"Undo");
+
+        /// <summary>
+        /// "Redo"
+        /// </summary>
+        public static LocalisableString Redo => new TranslatableString(getKey(@"redo"), @"Redo");
+
+        /// <summary>
+        /// "Cut"
+        /// </summary>
+        public static LocalisableString Cut => new TranslatableString(getKey(@"cut"), @"Cut");
+
+        /// <summary>
+        /// "Copy"
+        /// </summary>
+        public static LocalisableString Copy => new TranslatableString(getKey(@"copy"), @"Copy");
+
+        /// <summary>
+        /// "Paste"
+        /// </summary>
+        public static LocalisableString Paste => new TranslatableString(getKey(@"paste"), @"Paste");
+
+        /// <summary>
+        /// "Clone"
+        /// </summary>
+        public static LocalisableString Clone => new TranslatableString(getKey(@"clone"), @"Clone");
+
+        /// <summary>
+        /// "View"
+        /// </summary>
+        public static LocalisableString View => new TranslatableString(getKey(@"view"), @"View");
```

Comment:
These are all pretty common toolbar strings. May be better placed in `CommonStrings`.